### ISSUE TITLE
Question about pretrianed model

### ISSUE THREAD

**Issue by haopo2005** [PRBonn/bonnet#29]
Hi,
1.
I'd like to know the training details about your pretrained model on cityspace dataset.
Are all the hyper-parameters and training tricks inside the data.yaml, train.yaml, and net.yaml？
e.g. 
mutil-scale training, data augment, ...
Can you show me the training loss graph?

2.
As for other similiar semantic labeling dataset, e.g. raining, night, foggy scenes
If they follow the cityspace labeling format, Can I expand the training set and continue to train based on your per-trained model?
